### PR TITLE
Correct documentation on use Vuex Modules with vue-test-utils

### PR DESCRIPTION
**Pull request by gplusdotgr**

### CODE REVIEW

**Comment by eddyerburgh** [comment]
Thanks @gkatsanos :)

This example has a repository with the full code. Would you be able to make a PR against that to keep it in sync with your changes (e.g. changing the component from Modules to MyComponent).

https://github.com/eddyerburgh/vue-test-utils-vuex-example

**Comment by gplusdotgr** [comment]
done. Could someone check that what i did makes sense? I did a modification to the assertion test and I'm unsure it's correct.

File: docs/en/guides/using-with-vuex.md [hunk 1]
```diff
@@ -240,23 +240,27 @@ describe('Modules.vue', () => {
...
     button.trigger('click')
     expect(actions.moduleActionClick).toHaveBeenCalled()
   })
 
   it('Renders "state.inputValue" in first p tag', () => {
-    const wrapper = shallowMount(Modules, { store, localVue })
+    const wrapper = shallowMount(MyComponent, { store, localVue })
     const p = wrapper.find('p')
-    expect(p.text()).toBe(state.module.clicks.toString())
+    expect(p.text()).toBe(state.clicks.toString())
```
Comment (gplusdotgr) [inline docs/en/guides/using-with-vuex.md:263]: @eddyerburgh I guess that's wrong right?

Comment (eddyerburgh) [reply docs/en/guides/using-with-vuex.md:263]: Yes, can you change it in the example repo too please? https://github.com/eddyerburgh/vue-test-utils-vuex-example/blob/master/test/unit/specs/MyComponent.spec.js#L47

Comment (gplusdotgr) [reply docs/en/guides/using-with-vuex.md:263]: wait, so my change is correct? not sure

Comment (eddyerburgh) [reply docs/en/guides/using-with-vuex.md:263]: Yes your change is correct

Comment (gplusdotgr) [reply docs/en/guides/using-with-vuex.md:263]: https://github.com/eddyerburgh/vue-test-utils-vuex-example/pull/10

**Comment by gplusdotgr** [comment]
https://github.com/eddyerburgh/vue-test-utils-vuex-example/pull/10

**Comment by eddyerburgh** [comment]
Thanks George. We've changed the docs structure since this PR was created, can you merge and add your changes to the new docs/guides/using-with-vuex.md?

**Comment by gplusdotgr** [comment]
I'm getting merge conflicts in package.json for some reason.

**Comment by 38elements** [comment]
If  it  is your difficulty  resolving conflicts, I think it is easier to close this pull request and add changes to latest repository and send new pull request.

**Comment by eddyerburgh** [comment]
I've merged the changes in a new PR, thanks for your work here :)

**Pull request closed without merging**